src/main/java/Kowalski.java

```diff
@@ -0,0 +1,255 @@
+     * @throws KowalskiException In the event that the input has no "/by"
+     */
+    public static void checkDeadlineInput(String deadlineDetails) throws KowalskiException{
+        if (!(deadlineDetails.contains("/by"))) {
+            throw new KowalskiException();
+        }
+    }
+
+    /**
+     * Used to check if the user has accurately input the deadline by stating the "/from" and "/to"
+     * @param eventDetails : Contains the details of the event task
+     * @throws KowalskiException In the event that the input has no "/from" or "/to" or both
+     */
+    public static void checkEventInput(String eventDetails) throws KowalskiException{
+        if (!((eventDetails.contains("/from")) && (eventDetails.contains("/to")))) {
+            throw new KowalskiException();
+        }
+    }
+
+    /**
+     * Used to process the different variations of the users inputs
+     * @param userInput : String which the user inputs
+     * @return String which is in lowercase and clear of any unnecessary whitespace
+     */
+    public static String processInput(String userInput){
+        return (userInput.trim()).toLowerCase();
+    }
+
+    /**
+     * Prints out an accurate message for the number of tasks in the list.
+     * @param number : represents the total current task count
+     */
+    public static void printCurrentTaskMessage(int number){
+        switch (number){
+        case 0:
+            System.out.println("Now you have 0 tasks in the list.");
+            break;
+        case 1:
+            System.out.println("Now you have 1 task in the list.");
+            break;
+        default:
+            System.out.println("Now you have " + number + " tasks in the list.");
+        }
+    }
+
+    /**
+     * Used in the "list" command to print all the Current Tasks in the proper format
+     */
+    public static void printCurrentTaskItems(){
+        for (int i = 1; i <= currentTask.size(); i++){
+            System.out.println(i + "." + currentTask.get(i-1));
+        }
+    }
+
+    /**
+     * Cleans up the user input and forms new Deadline Task
+     * @param deadlineDetails : User input for details of the deadline task
+     * @return new deadline task created
+     */
+    private static Task getNewDeadlineTask(String deadlineDetails) {
+        String[] deadlineArray = deadlineDetails.split("/by");
+        for (int i = 0; i < deadlineArray.length; i++) {
+            deadlineArray[i] = deadlineArray[i].trim();
+        }
+
+
+        return new Deadline(deadlineArray[0], deadlineArray[1]);
+    }
+
+
+    /**
+     * Cleans up the user input and forms new event Task
+     * @param eventDetails : User input for details of the event task
+     * @return new event task created
+     */
+    private static Task getNewEventTask(String eventDetails) {
+        String[] eventArray = eventDetails.split("/from");
+        String eventInformation = eventArray[0].trim();
+        String [] fromAndTo = eventArray[1].split("/to");
```

Comment:
array specifiers should be attached to the type 
https://se-education.org/guides/conventions/java/basic.html#types